<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEvent</class>
 <widget class="QDialog" name="EditEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>New time:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="minimumDate">
      <date>
       <year>2021</year>
       <month>6</month>
       <day>5</day>
      </date>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>New notes:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>New date:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineNewHeader"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>New header:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>New frequency:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>One time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Every hour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Every day</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Every week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Every month</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
